Gemfile

```diff
@@ -10,6 +10,7 @@ gem 'rails-observers'
 # Models
 gem 'active_model_serializers'
 gem 'chronic'
+gem 'gutentag', '~> 2.6'
```

Comment:
I'm introducing [this gem](https://github.com/pat/gutentag) as a new dependency. It seems relatively well maintained to me, and is tested against Ruby 3 and Rails 6.1+, so I don't anticipate it being much of a drag to maintain.